test/acceptance/p2p/host/host_integration_test.cpp

```diff
@@ -120,5 +121,7 @@ namespace {
 INSTANTIATE_TEST_CASE_P(AllTestCases, HostIntegrationTest,
                         ::testing::Values(
                             // ports are not freed, so new ports each time
-                            Config{1u, 1u, 40510u, 2s, 2s, 200ms},
-                            Config{2u, 1u, 40510u, 2s, 2s, 200ms}));
+                            Config{1u, 1u, 40510u, 2s, 2s, 200ms, false},
+                            Config{2u, 1u, 40510u, 2s, 2s, 200ms, false},
+                            Config{1u, 1u, 40510u, 2s, 2s, 200ms, true},
+                            Config{2u, 1u, 40510u, 2s, 2s, 200ms, true}));
```

Comment:
[optional, not critical]. GTest can generate cartesian product of params using Combine. [link](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#how-to-write-value-parameterized-tests). You can use it to avoid copying last two `Config`s differentiating them only by the last bool value. I guess something like:
```
Combine(Values(Config{1u, 1u, 40510u, 2s, 2s, 200ms},
               Config{2u, 1u, 40510u, 2s, 2s, 200ms}), 
        Bool());
```
should work

Reply:
Thanks! I will try to apply this next time. (Here we have a relatively small quantity of cases to refactor this place - 4)